onnxruntime/python/tools/transformers/models/llama/llama_parity.py

```diff
@@ -66,21 +66,28 @@ def get_inputs(args: argparse.Namespace, config: AutoConfig):
     return inputs
 
 
-def verify_parity(
-    args: argparse.Namespace, config: AutoConfig, pt_model: AutoModelForCausalLM, kv_cache_ortvalues: dict
-):
+def verify_parity(args: argparse.Namespace, location, use_auth_token, kv_cache_ortvalues: dict):
+    config, llama = setup_torch_model(
```

Comment:
Setting up the PyTorch model in `verify_parity` will cause the PyTorch model to be loaded twice into memory because `verify_parity` is called twice. It is set up outside of `verify_parity` to avoid this and reduce time.

https://github.com/microsoft/onnxruntime/blob/eb0ce86db8eaa5a207dbd433ba8d0630dc60db4c/onnxruntime/python/tools/transformers/models/llama/llama_parity.py#L262-L268

Reply:
Thank your reminder. I added a parameter to load the model every time for CI. By default, it could be loaded only once.